[Spambayes] timtest broke?
After my latest cvs up, timtest fails with

    Traceback (most recent call last):
      File "/home/skip/src/spambayes/timtest.py", line 294, in ?
        drive(nsets)
      File "/home/skip/src/spambayes/timtest.py", line 264, in drive
        d = Driver()
      File "/home/skip/src/spambayes/timtest.py", line 152, in __init__
        self.global_ham_hist = Hist(options.nbuckets)
    AttributeError: 'OptionsClass' object has no attribute 'nbuckets'

I'm running it as

    timtest -n5 > Data/timtest.out

from my ~/Mail directory (not from my ~/src/spambayes directory).  If I
create a symlink to ~/src/spambayes/bayes.ini it works once again, but
shouldn't there be an nbuckets attribute with a default value already?

Skip

